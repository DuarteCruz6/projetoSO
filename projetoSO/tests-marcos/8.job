WRITE [(a4,35)(f2,84)(r5,29)(r5,17)(q1,51)]
WRITE [(u3,17)(v3,46)(s2,91)(n8,48)]
WRITE [(h5,83)]
WRITE [(j1,7)(h1,4)(g1,33)]
WRITE [(e7,94)(q8,5)(z0,71)(e5,16)]
READ [q1]
WRITE [(e2,4)(w0,84)(d4,23)(w3,33)(m2,15)]
READ [s2]
DELETE [h1,h5,g1,j1]
WRITE [(o4,17)]
WRITE [(n1,45)]
READ [q8]
WRITE [(w1,96)(m7,5)(v9,32)(l9,83)]
WAIT 3000
READ [r5]
READ [w1]
WRITE [(k8,21)(u2,41)(i7,44)]
BACKUP
