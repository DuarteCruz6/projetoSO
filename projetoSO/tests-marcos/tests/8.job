WRITE [(c4,42)(u4,85)(g5,40)(v6,37)]
WRITE [(f2,90)(p3,91)]
DELETE [g5,f2,p3,c4,u4]
WRITE [(y5,52)(b0,42)]
DELETE [b0,v6]
READ [y5]
WRITE [(i1,5)(c5,25)(i4,21)(s6,95)(w8,23)]
WRITE [(n9,30)]
DELETE [n9,c5,s6,w8,y5]
BACKUP
READ [i4]
WRITE [(i2,14)(d8,65)(k3,44)]
WRITE [(m3,64)(q6,92)(q5,10)]
WRITE [(u9,19)(p9,78)(c3,41)]
DELETE [k3,p9,i1,c3,q6]
WRITE [(v6,45)(c6,33)(o2,46)(u3,49)]
WRITE [(w9,80)(q9,8)(p6,9)(f1,13)(g3,73)]
WRITE [(p9,96)(h6,86)(p0,13)]
READ [o2]
DELETE [u3,m3]
WRITE [(w8,15)]
WRITE [(u5,60)(t6,9)]
WRITE [(a6,48)(l1,21)(b0,1)]
WRITE [(l0,82)(b5,64)(l4,62)(y0,87)(p7,6)]
DELETE [i4,q5,l1,b5]
DELETE [i2]
WRITE [(b7,58)]
WRITE [(e5,50)(w4,60)(x5,5)(w1,5)]
WRITE [(x4,28)(v3,45)(f3,38)(i9,97)]
READ [h6]
DELETE [p6,p0,x4,u5]
READ [x5]
WRITE [(h3,60)(t5,68)(o5,84)(t0,98)(z2,73)]
READ [b7]
WAIT 2000
WRITE [(s8,35)(u1,25)(f7,51)(f0,57)]
SHOW
WRITE [(w7,96)]
WRITE [(e8,19)(e4,46)(z3,83)(q6,19)]
WRITE [(t3,93)(w3,93)(u8,86)(l2,46)(y9,77)]
WRITE [(p1,82)(p0,96)(s7,88)(n7,100)(o4,85)]
WAIT 2000
WAIT 1000
WRITE [(k5,79)]
BACKUP
READ [y9]
DELETE [x5,g3,u9,k5]
WRITE [(q5,66)(g2,8)]
SHOW
WRITE [(b8,6)(k5,13)(n9,0)]
WRITE [(a7,40)]
BACKUP
DELETE [b0,w3]
WRITE [(p3,37)(b0,25)]
WRITE [(m8,56)(q0,83)(k3,83)(n8,58)]
WRITE [(u6,8)]
WRITE [(p8,99)(c1,78)(o8,22)(c2,31)(i8,34)]
WRITE [(q2,86)(e1,83)(u4,95)(a3,19)]
WRITE [(m1,88)(t9,56)(g1,25)]
READ [h6]
WAIT 1000
DELETE [w1]
WAIT 1000
WRITE [(j9,97)(x8,85)(b6,49)]
READ [t5]
