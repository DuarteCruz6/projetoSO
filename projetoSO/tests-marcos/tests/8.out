[(y5,52)]
[(i4,21)]
[(o2,46)]
[(h6,86)]
[(x5,5)]
[(b7,58)]
(a6, 48)
(b7, 58)
(b0, 1)
(c6, 33)
(d8, 65)
(e5, 50)
(f0, 57)
(f7, 51)
(f3, 38)
(f1, 13)
(g3, 73)
(h3, 60)
(h6, 86)
(i9, 97)
(l4, 62)
(l0, 82)
(o5, 84)
(o2, 46)
(p7, 6)
(p9, 96)
(q9, 8)
(s8, 35)
(t0, 98)
(t5, 68)
(t6, 9)
(u1, 25)
(u9, 19)
(v3, 45)
(v6, 45)
(w1, 5)
(w4, 60)
(w8, 15)
(w9, 80)
(x5, 5)
(y0, 87)
(z2, 73)
[(y9,77)]
(a6, 48)
(b7, 58)
(b0, 1)
(c6, 33)
(d8, 65)
(e4, 46)
(e8, 19)
(e5, 50)
(f0, 57)
(f7, 51)
(f3, 38)
(f1, 13)
(g2, 8)
(h3, 60)
(h6, 86)
(i9, 97)
(l2, 46)
(l4, 62)
(l0, 82)
(n7, 100)
(o4, 85)
(o5, 84)
(o2, 46)
(p0, 96)
(p1, 82)
(p7, 6)
(p9, 96)
(q5, 66)
(q6, 19)
(q9, 8)
(s7, 88)
(s8, 35)
(t3, 93)
(t0, 98)
(t5, 68)
(t6, 9)
(u8, 86)
(u1, 25)
(v3, 45)
(v6, 45)
(w3, 93)
(w7, 96)
(w1, 5)
(w4, 60)
(w8, 15)
(w9, 80)
(y9, 77)
(y0, 87)
(z3, 83)
(z2, 73)
[(h6,86)]
[(t5,68)]
